src/Swipeable.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './Swipeable.css';
+import PropTypes from 'prop-types';
+
+// A card that can be swiped left or right, handling those animations and interactions
+// and calling back to the props when either occurs.
+// TODO: update with react-swipeable-views
```

Comment:
This is where the swiping interaction will go!

http://oliviertassinari.github.io/react-swipeable-views/ should help :)